modules/RockRuby.cmake

```diff
@@ -282,12 +292,10 @@ ELSEIF(NOT RUBY_EXTENSIONS_AVAILABLE)
         target_link_libraries(${target} ${RUBY_LIBRARY})
 
         STRING(REGEX MATCH "arm.*" ARCH ${CMAKE_SYSTEM_PROCESSOR})
-        IF(NOT APPLE)
-            IF("${ARCH}" STREQUAL "")
-                set_target_properties(${target} PROPERTIES
-                    LINK_FLAGS "-z noexecstack")
-            ENDIF("${ARCH}" STREQUAL "")
-        ENDIF(NOT APPLE)
+        IF("${ARCH}" STREQUAL "")
+            set_target_properties(${target} PROPERTIES
+                LINK_FLAGS "-Wl,-z,noexecstack")
```

Comment:
??? -Wl is the compiler flag, LINK_FLAGS set link flags. Is the -Wl, really required ? Was anyone hitting that line then (because if there was an error, nobody mentioned it so far). What am I missing ???